src/errors/configurationError.ts

```diff
@@ -6,13 +6,16 @@ type ConfigurationErrorHandler = (error: ConfigurationErrorOptions) => string;
 /** Explains Resolution Error options */
 type ConfigurationErrorOptions = {
   method?: ResolutionMethod;
+  dependacy?: string;
+  version?: string;
 };
 
 export enum ConfigurationErrorCode {
   IncorrectProvider = 'IncorrectProvider',
   UnspecifiedNetwork = 'UnspecifiedNetwork',
   UnspecifiedUrl = 'UnspecifiedUrl',
   MissingProviderConfigurations = 'MissingProviderConfigurations',
+  DependacyMissing = "DependacyMissing"
```

Comment:
Please fix the typo in all places:

```suggestion
  DependancyMissing = "DependancyMissing"
```

Reply:
It's actually depend**E**ncy. :)

Reply:
The time has come. I need Grammarly in coding...